src/Serializer/AbstractSerializer.php

```diff
@@ -210,7 +210,7 @@ protected function serializeValue($value)
      *
      * @return string
      */
-    protected function serializeCallable(callable $callable): string
+    protected function serializeCallable($callable): string
```

Comment:
As pointed out by @ste93cry in https://github.com/getsentry/sentry-php/pull/821#issuecomment-495261003, this is a BC, we cannot remove the type hint...